ansible/pulp-from-source.yml

```diff
@@ -24,11 +24,5 @@
     - lazy
     - dev
     - {role: plugin, plugin_name: 'pulp_file', app_label: 'pulp_file'}
-    # - {role: plugin, plugin_name: 'pulp_example', app_label: 'pulp_example'}
```

Comment:
Please leave this one. It would be reasonable to change it to ```pulp_python``` though, since we're getting rid of pulp_example.

It's nice to just be able to just uncomment the plugins you want to install.

Reply:
Sure. I'll do that.